recipes/qxlsx/all/conanfile.py

```diff
@@ -52,12 +52,16 @@ def source(self):
     @functools.lru_cache(1)
     def _configure_cmake(self):
         cmake = CMake(self)
+        cmake.definitions["QT_VERSION_MAJOR"] = tools.Version(self.deps_cpp_info["qt"].version).major
         cmake.configure()
         return cmake
 
     def build(self):
         for patch in self.conan_data.get("patches", {}).get(self.version, []):
             tools.patch(**patch)
+        tools.replace_in_file(os.path.join(self._source_subfolder, "QXlsx", "CMakeLists.txt"),
```

Comment:
Could this be option, only if qt6 set the the required cxxstd? would be a fun surprise for qt5 consumers

Reply:
Done here: 798492233.

Reply:
Sorry the commit had to be fixed here: 329ec4c7a